sql/api/src/main/scala/org/apache/spark/sql/catalyst/util/QuotingUtils.scala

```diff
@@ -37,6 +49,18 @@ object QuotingUtils {
     }
   }
 
+  def quoted(namespace: Array[String]): String = {
+    namespace.map(quoteIfNeeded).mkString(".")
+  }
+
+  def quoted(ident: Identifier): String = {
```

Comment:
Nit: I am wondering if this should be renamed to `quotedIdentifier`. Same for the above which might be `quotedNamespace`?

Reply:
It may be easier to use if quoted supports overload?

Reply:
`quoted` may be easier to use because of the overload support?

Reply:
I am fine with either way